### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by samlandfried**
DB added and connected

### CODE REVIEW

File: db/seeds/dev/meal_foods.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+const pry = require('pryjs')
```
Comment (samlandfried) [inline db/seeds/dev/meal_foods.js]: @tmikeschu Seeding the DB works about 50% of the time. The other half of the time I get this error: `error: insert or update on table "meal_foods" violates foreign key constraint "meal_foods_food_id_fkey"` I think maybe my scheme for creating foreign keys is somehow creating values out of range? I've exhausted my troubleshooting abilities, though, so please lemme know if you see what's up or have any ideas

File: test/server-test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  it('Should exist', () => {
+    assert.ok(app);
+  });
+
+  describe('GET', () => {
+    it('Should redirect root requests to the Github repo', () => {
+      request.get('http://localhost:9876', (error, response) => {
+        if (error) { done(error); }
+        console.log(response);
```
Comment (samlandfried) [inline test/server-test.js]: @tmikeschu All the assertions I put in this callback (Like on line 3) don't run, but the `console.log` here does run. I imagine I'm handling async incorrectly. Do you see where I'm going wrong?

File: db/seeds/dev/meal_foods.js (hunk 1 above)
Comment (tmikeschu) [inline db/seeds/dev/meal_foods.js]: I think you're right about the fkey being out of range. I would debug/pry to see what your collection of ID numbers are, and make sure they are `1..x` every time. You might also try a different way to populate numbers.

File: test/server-test.js (hunk 2 above)
Comment (tmikeschu) [inline test/server-test.js]: You have to pass `done` in your `it` line. `it('', done => {`

File: db/seeds/dev/meal_foods.js (hunk 1 above)
Comment (samlandfried) [inline db/seeds/dev/meal_foods.js]: Cool, thanks - I figured out promises don't get executed in order (At least when being invoked w/ `Promise.all`) so sometimes the corresponding food or meal hadn't been created by the time it tried to create the joins table entry

File: test/server-test.js (hunk 2 above)
Comment (samlandfried) [inline test/server-test.js]: Yep, that was it. Thanks!

File: db/seeds/dev/meal_foods.js (hunk 1 above)
Comment (tmikeschu) [inline db/seeds/dev/meal_foods.js]: Nice! I was going to suggest that, but I thought Promise.all executed in the order their passed in, so your `concat` sequence looked solid. Nice fix!

**Pull request merged**